pageserver/src/tenant/timeline.rs

```diff
@@ -1958,6 +1947,7 @@ impl Timeline {
             .unwrap_or(self.conf.default_tenant_conf.lsn_lease_length_for_ts)
     }
 
+    #[allow(dead_code)]
```

Comment:
Why do you not remove this function? Ideally there would be a comment that explains it.

Reply:
I want to leave the config items here until the release after the next release, will add a comment